homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -59,6 +71,10 @@ def see_device(address, name, new_device=False):
                     return
                 _LOGGER.debug("Adding %s to tracked devices", address)
                 devs_to_track.append(address)
+                if battery_track_interval > timedelta(0):
+                    devs_track_battery.update(
```

Comment:
```suggestion
                    devs_track_battery[address] = dt_util.as_utc(datetime.fromtimestamp(0))
```

No need to do update if its only one key/value